config/src/main/java/org/springframework/security/config/annotation/web/builders/FilterComparator.java

```diff
@@ -150,6 +150,23 @@ public void registerAfter(Class<? extends Filter> filter,
 	}
 
 	/**
+	 * Registers a {@link Filter} to exist at a particular {@link Filter} position
+	 * @param filter the {@link Filter} to register
+	 * @param atFilter the {@link Filter} that is already registered and that
+	 * {@code filter} should be placed at.
+	 */
+	public void registerAt(Class<? extends Filter> filter,
```

Comment:
Yes, but this is up to the users discretion. This is no different than a user stating to put a key / value pair in a Map if the key already exists